### ISSUE TITLE
Crazy idea of an AI binary bot 

### ISSUE THREAD

**Issue by christiangreat** [binary-com/binary-bot#3966]
Hello, there’s this idea of a binary bot. The basic idea of it, is to Gather information-Learn-Create a formula-Execute. 

So after my research on synthetic indices, I learnt that The movement of synthetic indices is accomplished by the use of random numbers that are produced by a computer program. And, the value of the synthetic indices is generated by the algorithm, and it is directed by the types of market situations that the indices are intended to replicate.

So I don’t know how possible it is, to create a bot that scans the historical data of a specific synthetic indices, let’s say Vix 75. Analyses the chart patterns, then create a mathematical formula, that matches the algorithm of it. Then use this formula to enter and close trades. 

It’s this possible to achieve?

**Comment by binary-deriv-master** [comment]
Hello friend @christiangreat  , you can see all the historical data in the tutorials in our communities about that 

We remind you that Neuronal Trader only has 3 groups:

✅ (SPANISH GROUP)
https://t.me/neuronaltrader_comunidad
✅ (ENGLISH GROUP)
https://t.me/neuronaltrader_community
✅ (PORTUGUESE GROUP)
https://t.me/neuronaltrader_portugues

and there is only one channel for informational purposes:

👉(THE ONLY NEURONAL TRADER CHANNEL)👈
https://t.me/neuronaltrader

🔉🔊❕These are the official Neuronal Trader communities❕📢📣

**Comment by christiangreat** [comment]
Ok. Thank you. Please I need help with this. @binary-deriv-master 
[E-TECH GHOST RIDER V.002 (Scalper).zip](https://github.com/binary-com/binary-bot/files/13254493/E-TECH.GHOST.RIDER.V.002.Scalper.zip)

I borrowed some ideas from the larry Williams bot and I was thinking if you could help me add it to this bot and then make a few adjustments. 

Firstly, to add the Williams %R indicator with the parameter -20 to 0 overbought and then -100 to -80 oversold. 

Then add a stochastic indicator: 80 to 100 overbought, then 0 to 20 oversold. The parameters of the indicator will be K- 21, D- 3, smooth- 3

So to place a buy trade the confirmation remains same, just that the stochastic indicator and Williams %R indicator must be oversold, whereas to sell, the stochastic indicator and the Williams %R indicator must be overbought. 

Then the trades should open at the start of the second candle, instead of the third. 

Then the 25, 50, 100 ema to identify the trend remains same. 

Finally virtual hook should be added to it. 

In summary
So in an uptrend for instance in order to place a buy trade, the following must be met:
- The Williams %R and stochastic indicator must be oversold. 
- The Ema must identify it be an uptrend 
- There must be one green candle, after which the trade opens at the start of the next candle. 

In a downtrend for instance in order to place a sell trade, the following must be met:
- The Williams %R and stochastic indicator must be overbought. 
- The Ema must identify it be a downtrend 
- There must be one red candle, after which the trade opens at the start of the next candle. 

I think every other thing in this bot, should be same asides what I mentioned.

Thank you.

**Comment by jadersonk** [comment]
> Ok. Thank you. Please I need help with this. @binary-deriv-master [E-TECH GHOST RIDER V.002 (Scalper).zip](https://github.com/binary-com/binary-bot/files/13254493/E-TECH.GHOST.RIDER.V.002.Scalper.zip)
> 
> I borrowed some ideas from the larry Williams bot and I was thinking if you could help me add it to this bot and then make a few adjustments.
> 
> Firstly, to add the Williams %R indicator with the parameter -20 to 0 overbought and then -100 to -80 oversold.
> 
> Then add a stochastic indicator: 80 to 100 overbought, then 0 to 20 oversold. The parameters of the indicator will be K- 21, D- 3, smooth- 3
> 
> So to place a buy trade the confirmation remains same, just that the stochastic indicator and Williams %R indicator must be oversold, whereas to sell, the stochastic indicator and the Williams %R indicator must be overbought.
> 
> Then the trades should open at the start of the second candle, instead of the third.
> 
> Then the 25, 50, 100 ema to identify the trend remains same.
> 
> Finally virtual hook should be added to it.
> 
> In summary So in an uptrend for instance in order to place a buy trade, the following must be met:
> 
> * The Williams %R and stochastic indicator must be oversold.
> * The Ema must identify it be an uptrend
> * There must be one green candle, after which the trade opens at the start of the next candle.
> 
> In a downtrend for instance in order to place a sell trade, the following must be met:
> 
> * The Williams %R and stochastic indicator must be overbought.
> * The Ema must identify it be a downtrend
> * There must be one red candle, after which the trade opens at the start of the next candle.
> 
> I think every other thing in this bot, should be same asides what I mentioned.
> 
> Thank you.

It's serious, that you are in the official Deriv group discussing 'bout a bot how to extract money from the broker' using AI and exposes your formulas?
Look let me save a lot of headache, money and frustration:  first of all this is not possible with simples computer resources, and without a team of engineering. The amount of effort, time and money to does this It is absurd.
Second, that you will certainly be breaking some user terms compliance rules ( have you read It, at least a one time?), running the risk of being penalized, to spended a lot of money with professional team, to ended with no account, and less money that before and a bigger frustration.
The correct It is you does manual trading, analyzing the volatility market and weigh your risk of loss/gain. Participate from study group to discover new opportunities, could be increase your profits.

**Comment by christiangreat** [comment]
Ok, no problem. But is the second request I made possible?

**Comment by christiangreat** [comment]
> > Ok. Thank you. Please I need help with this. @binary-deriv-master [E-TECH GHOST RIDER V.002 (Scalper).zip](https://github.com/binary-com/binary-bot/files/13254493/E-TECH.GHOST.RIDER.V.002.Scalper.zip)
> > I borrowed some ideas from the larry Williams bot and I was thinking if you could help me add it to this bot and then make a few adjustments.
> > Firstly, to add the Williams %R indicator with the parameter -20 to 0 overbought and then -100 to -80 oversold.
> > Then add a stochastic indicator: 80 to 100 overbought, then 0 to 20 oversold. The parameters of the indicator will be K- 21, D- 3, smooth- 3
> > So to place a buy trade the confirmation remains same, just that the stochastic indicator and Williams %R indicator must be oversold, whereas to sell, the stochastic indicator and the Williams %R indicator must be overbought.
> > Then the trades should open at the start of the second candle, instead of the third.
> > Then the 25, 50, 100 ema to identify the trend remains same.
> > Finally virtual hook should be added to it.
> > In summary So in an uptrend for instance in order to place a buy trade, the following must be met:
> > 
> > * The Williams %R and stochastic indicator must be oversold.
> > * The Ema must identify it be an uptrend
> > * There must be one green candle, after which the trade opens at the start of the next candle.
> > 
> > In a downtrend for instance in order to place a sell trade, the following must be met:
> > 
> > * The Williams %R and stochastic indicator must be overbought.
> > * The Ema must identify it be a downtrend
> > * There must be one red candle, after which the trade opens at the start of the next candle.
> > 
> > I think every other thing in this bot, should be same asides what I mentioned.
> > Thank you.
> 
> It's serious, that you are in the official Deriv group discussing 'bout a bot how to extract money from the broker' using AI and exposes your formulas? Look let me save a lot of headache, money and frustration: first of all this is not possible with simples computer resources, and without a team of engineering. The amount of effort, time and money to does this It is absurd. Second, that you will certainly be breaking some user terms compliance rules ( have you read It, at least a one time?), running the risk of being penalized, to spended a lot of money with professional team, to ended with no account, and less money that before and a bigger frustration. The correct It is you does manual trading, analyzing the volatility market and weigh your risk of loss/gain. Participate from study group to discover new opportunities, could be increase your profits.

ok, noted. Thank you. But is my second request possible?

**Comment by binary-ex-machina** [comment]
@christiangreat, it will be tough to create such a bot because of the amount of data you can get from Deriv market data. Another issue here is applying models to learn the algorithm that Deriv uses to generate the ticks. It must be a robust model with a lot of data. Deriv changes the volatility daily, so, as an example, a strategy that uses a barrier of +1 with the One Touch trade and gives a 40% payout in one day. 

The next day, that same configuration may provide a 30% payout. It's impossible to train a bot to do that since it will adapt to the day's volatility only after the day ends. 

Applying machine learning on Deriv synthetic indexes is not worth it since it's a randomly generated market with a random volatility parameter. It is better to use that for Forex and commodities markets.

https://binaryblocks.pro/

![Screenshot 2023-10-25 at 13 35 02](https://github.com/binary-com/binary-bot/assets/34610787/1f77e711-fbd0-4572-a49a-45b958b475e8)
